launch/tier4_perception_launch/launch/object_recognition/detection/detection.launch.xml

```diff
@@ -6,6 +6,7 @@
   <!-- Pipeline junctions -->
   <arg name="mode" description="options: `camera_lidar_radar_fusion`, `camera_lidar_fusion`, `lidar_radar_fusion`, `lidar` or `radar`"/>
   <arg name="lidar_detection_model" description="options: `transfusion`, `centerpoint`, `apollo`, `pointpainting`, `clustering`"/>
+  <arg name="bev_detection_model" default="bevdet" description="options: `bevdet`"/>
```

Comment:
I think we can also remove this args for now, as we only have one single choice for now (just launch tensorrt_bevdet by default).

Of course I understand that this change is reasonable if we are to add bunch of BEV based method in the future, but we aren't sure if that's going to happen in a short period of time. Instead, I want to make the launcher as simple as possible. We can leave this task to the one who is going to implement the second BEV based method if any.